src/transformers/models/llava_next/convert_llava_next_weights_to_hf.py

```diff
@@ -305,16 +364,27 @@ def convert_llava_to_hf(model_id, pytorch_dump_folder_path, push_to_hub=False):
     outputs = tokenizer.batch_decode(output_ids, skip_special_tokens=True)
     print(outputs)
 
-    if pytorch_dump_folder_path is not None:
-        print(f"Saving model and processor for {model_id} to {pytorch_dump_folder_path}")
-        Path(pytorch_dump_folder_path).mkdir(exist_ok=True)
-        model.save_pretrained(pytorch_dump_folder_path)
-        processor.save_pretrained(pytorch_dump_folder_path)
+    import os
+
+    from huggingface_hub import HfApi
+
+    repo_id = model_id.split("/")[-1]
```

Comment:
Let's use a better name here - this isn't really the "repo_id", it's the checkpoint name